api/core.py

```diff
@@ -416,7 +430,11 @@ def _archivestream(self, ticket):
         stream = cStringIO.StringIO()
         with tarfile.open(mode='w|', fileobj=stream) as archive:
             for filepath, arcpath, _ in ticket['target']:
-                yield archive.gettarinfo(filepath, arcpath).tobuf()
+                try:
+                    yield archive.gettarinfo(filepath, arcpath).tobuf()
+                except Exception as e:
```

Comment:
It is not possible to have an incomplete tarball.
If there is an error on a file, it is fully discarded.

I understand your concern on discarding exceptions, but I thought that if there is an error on a single file for any reason it is still better to send the other files in any case. Should we make a distinction between a corrupted and a missing file in the user response?

Also, we are not completely blind to those errors, as we are logging a warn for all of them.